front/src/modules/trainschedule/components/DriverTrainScheduleV2/DriverTrainScheduleStopListV2.tsx

```diff
@@ -35,9 +35,9 @@ const DriverTrainScheduleStopListV2 = ({
 
   const formattedMrsp = useMemo(
     () =>
-      mrsp.positions.map((_position, index) => ({
+      mrsp.boundaries.map((_position, index) => ({
```

Comment:
Logic to be explored, but at least the last boundary doesn't have a corresponding value
```suggestion
      mrsp.boundaries.slice(0,-1).map((_position, index) => ({
```
Same on `exportDriverScheduleCSV.ts` below

Thanks @RomainValls